src/atoms.jl

```diff
@@ -160,3 +201,4 @@ end
 save(file::Union{String,IOStream}, system::Atoms; kwargs...) = write_frame(file, write_dict(system); kwargs...)
 
 save(file::Union{String,IOStream}, systems::Vector{Atoms{NT1,NT2}}; kwargs...) where {NT1,NT2} = write_frames(file, write_dict.(systems); kwargs...)
+save(file::Union{String,IOStream}, systems::Vector{Atoms}; kwargs...) = write_frames(file, write_dict.(systems); kwargs...)
```

Comment:
I ran into a problem when i loaded the test.xyz file and tried to safe this.
Then this error occured:
> ERROR: MethodError: no method matching save(::String, ::Vector{Atoms})
Closest candidates are:
  save(::Union{IOStream, String}, ::Atoms; kwargs...) at C:\Users\janha\Documents\Programming\git\SEP\ExtXYZ.jl\src\atoms.jl:201
  save(::Union{IOStream, String}, ::Array{Atoms{NT1, NT2}, 1}; kwargs...) where {NT1, NT2} at C:\Users\janha\Documents\Programming\git\SEP\ExtXYZ.jl\src\atoms.jl:203
Stacktrace:
 [1] top-level scope
   @ REPL[5]:1

I do not fully understand the reason of this error, because in the tests everything seems to work, however with this extra method the error didn't occur.
Complete forgot that i changed this, sorry for no comment on this.

Reply:
OK, let's go with this.